HockeyAppForWindows/HockeyBootstrapper.cs

```diff
@@ -37,7 +38,9 @@ static HockeyBootstrapper()
         {
             LogManager.GetLog = type => new NLogLogger(type);
             HockeyApp.HockeyLogManager.GetLog = type => new HockeyAppLogger();
-            
+
+            // Set .Net to use only TLS 1.2 protocol
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Do we really want to force 1.2 on the client-side or do we want to rely on the server-side restrictions once we roll it back out?

Reply:
It work also with rink (which is TLS 1.1) - I think it use it, but the enforce to use **only** this is on the server-side

Reply:
I agree to what @meijin007 says. Rink allows TLS 1.0/1.1/1.2 and this is telling the client to use only 1.2.